src/main/java/org/jitsi/jigasi/transcription/LocalTxtTranscriptHandler.java

```diff
@@ -0,0 +1,413 @@
+/*
+ * Jigasi, the JItsi GAteway to SIP.
+ *
+ * Copyright @ 2017 Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.jigasi.transcription;
+
+import org.jitsi.util.*;
+
+import java.io.*;
+import java.time.*;
+import java.util.*;
+
+/**
+ * Handler for formatting a transcript as a .txt file and storing it locally
+ *
+ * @author Nik Vaessen
+ */
+public class LocalTxtTranscriptHandler
+    extends AbstractTranscriptHandler<String>
+{
+
+    /**
+     * The logger of this class
+     */
+    private static final Logger logger = Logger.getLogger(Transcript.class);
+
+    /**
+     * The root dir where a new directory will be created in to store
+     * transcripts
+     */
+    private static final String ROOT_DIR = System.getProperty("user.dir");
```

Comment:
Can you use the config service and a separate property name, e.g. org.jitsi.jigasi.transcription.LocalTxtTranscriptHandler.DIRECTORY (which can combine ROOT_DIR and LOG_DIR)?